examples/demo/src/layout/SubMenu.js

```diff
@@ -32,11 +32,12 @@ const SubMenu = ({
 }) => {
     const translate = useTranslate();
     const classes = useStyles();
+    const Icon = icon;
```

Comment:
You can simply rename it in the props instead of declaring yet another variable:
```js
const SubMenu = ({
    handleToggle,
    sidebarIsOpen,
    isOpen,
    name,
    icon: Icon,
    children,
}) => ...
```